{{cookiecutter.project_name}}/.github/workflows/run-tests.yml

```diff
@@ -20,6 +20,12 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.8
+       {%- if cookiecutter.include_feature_store == "yes" %}
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '11'
```

Comment:
I missed this originally, out of curiosity why do the feature store tests require Java? Should we be documenting the Java requirement in the ML developer guide for feature store e.g. in the prereqs for developing locally?

Reply:
Updated. Java is needed as we are creating a local Spark session for testing.